### PR TITLE
Added some logs

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
Added some useful logs, filters, hits file, to show:
```
... -verbosity:minimal
```
On my local I see issue with missing hits file, template tests.
image

### CODE REVIEW

**Comment by tonerdo** [comment]
Thanks @MarcoRossignoli, what do you think about automatically pre-pending `[coverlet]` to all verbose logs to it's easy to pick them out?

**Comment by MarcoRossignoli** [comment]
> Thanks @MarcoRossignoli, what do you think about automatically pre-pending `[coverlet]` to all verbose logs to it's easy to pick them out?

I thought about that...and I agree...I'll update ASAP

**Comment by MarcoRossignoli** [comment]
@tonerdo PTAL more clear!
image

File: src/coverlet.msbuild.tasks/MSBuildLogger.cs [hunk 1]
```diff
@@ -7,19 +7,21 @@ namespace Coverlet.MSbuild.Tasks
 {
     class MSBuildLogger : ILogger
     {
+        private const string LogPrefix = "[coverlet]";
```
Comment (tonerdo) [inline src/coverlet.msbuild.tasks/MSBuildLogger.cs]: Nit: Append space

Comment (MarcoRossignoli) [reply src/coverlet.msbuild.tasks/MSBuildLogger.cs]: done

Comment (MarcoRossignoli) [reply src/coverlet.msbuild.tasks/MSBuildLogger.cs]: image

**Pull request merged**